config/locales/en.yml

```diff
@@ -90,8 +90,8 @@ en:
         update: "This comment could not be modified because:"
       status:
         created_at: Posted on %{created_at}.
-        deleted_by: Comment deleted by %{deleter_name}.
-        updated_at: Last modified by %{editor_name} on %{updated_at}.
+        deleted_by: This comment has been deleted.
+        updated_at: Last modified by on %{updated_at}.
```

Comment:
not sure what this is for but: for `updated_at` should it be "Last modified on %{updated_at}."